internal/jptrace/attributes.go

```diff
@@ -18,13 +18,22 @@ const (
 	FormatAttribute = "@jaeger@format"
 )
 
-func AttributesToMap(attributes pcommon.Map) map[string]string {
-	tags := make(map[string]string)
+func AttributesToStringMap(attributes pcommon.Map) map[string]string {
+	mapAttributes := make(map[string]string)
 	attributes.Range(func(k string, v pcommon.Value) bool {
-		tags[k] = v.AsString()
+		mapAttributes[k] = v.AsString()
 		return true
 	})
-	return tags
+	return mapAttributes
+}
+
+func AttributesToMap(attributes pcommon.Map) map[string]any {
```

Comment:
why do you need this function as public? Doesn't look like any prod code is using it. If this is because of my previous review then I only meant it as a helper in the test itself (which doesn't look like you even need, you managed with AttributesToStringMap)

Reply:
@yurishkuro Removed